### ISSUE TITLE
Kontakt.io Beacon not showing up in Google's Beacon Tools unless "Secure Shuffling" enabled

### ISSUE THREAD

**Issue by koenschauwaert** [google/physical-web#930]
Kontakt.io Beacon not showing up in Google's Beacon Tools unless "Secure Shuffling" enabled.

I've got a Kontakt.io beacon which works very well, but only when I activate "Secure Shuffling" on the beacon itself:
https://support.kontakt.io/hc/en-gb/articles/206762009-Kontakt-io-Secure-Shuffling

When enabled I can see the beacon being unregistered and I am able to register the beacon and set a Nearby Notification via Google's Beacon Dashboard:
https://developers.google.com/beacons/dashboard/

But the problem with"Secure Shuffling" is that this beacon changes it's values after 24h. Therefore I have to re-register it and set up the beacon again.

Anyone encountering the same issues? How do I solve this? Why isn't my beacon showing up when "Secure Shuffling" is disabled?

**Comment by synergy3000** [comment]
Have you found any resolution to this?  I am having same problem.  When I manually enter the UUID it shows up but only as iBeacon.  I am trying to set it up with Eddystone.  I am using iOS with Beacon Tools.  Need to get an Android device to see if problem is same over there.  
Thanks

**Comment by koenschauwaert** [comment]
No, I still have no solid solution. Most of the time I have to open the Kontakt.io app on my Android device, Sync it with the cloud and then I can register the beacon in the Beacon Tools app by Google. After registering, it shows on my Google Beacon Dashboard. 
Sometimes I have to do this daily, other times it stays active for weeks.
